gulpfile.js

```diff
@@ -12,9 +27,78 @@ uswds.paths.dist.fonts = './public/fonts';
 uswds.paths.dist.img = './public/img';
 uswds.paths.dist.theme = './uswds';
 
+// Tasks to build USWDS assets for the Touchpoints app.
+// These tasks use the USWDS compiler.
 exports.copyJS = uswds.copyJS;
 exports.compileSass = uswds.compileSass;
 exports.copyFonts = uswds.copyFonts;
 exports.copyImages = uswds.copyImages;
 exports.compileIcons = uswds.compileIcons;
-exports.updateUswds = uswds.updateUswds;
\ No newline at end of file
+exports.updateUswds = uswds.updateUswds;
+
+// Tasks to build USWDS assets for the embedded survey widget.
+// These tasks do not use the USWDS compiler.
+exports.bundleWidgetJS = bundleWidgetJS;
+exports.compileWidgetSass = compileWidgetSass;
+exports.buildWidgetAssets = parallel(bundleWidgetJS, compileWidgetSass);
+
+// TODO: Do we need to babelify?
+async function bundleWidgetJS() {
+  return browserify("uswds/widget-uswds.js", {
+    paths: ['./node_modules'],
+    debug: true
+  })
+    .transform("aliasify", {
+      aliases: {
+        '../../uswds-core/src/js/config': './uswds/uswds-config.js'
+      },
+      verbose: true,
+      global: true
+    })
+    .bundle()
+    .pipe(source('touchpoints-uswds.js'))
+    .pipe(header("/* This file was generated by the gulp task 'bundleWidgetJS'. */\n\n"))
+    .pipe(buffer())
+    .pipe(dest(uswds.paths.dist.js))
+    .pipe(sourcemaps.init({ loadMaps: true }))
+    .pipe(uglify())
+    .pipe(rename({ suffix: ".min" }))
+    .pipe(sourcemaps.write("."))
+    .pipe(dest(uswds.paths.dist.js));
+}
+
+// TODO: Add browserslistrc
+async function compileWidgetSass() {
```

Comment:
Copied with minor changes from the [USWDS repository](https://github.com/uswds/uswds/blob/develop/tasks/sass.js). The main change is that I added a `prefixer` step which adds the `tp-` prefix to our version of the USWDS classes.